R/datasets.R

```diff
@@ -124,3 +124,27 @@ NULL
 #' @keywords data
 #' @docType data
 NULL
+
+
+#' Gunpoint functional data classification task.
```

Comment:
The prediction is whether the time series is about  a person pointing a gun or not, do I need to explain here